pkg/logging/scrubbingLogWriter.go

```diff
@@ -188,6 +189,13 @@ func addMandatoryMasking(dict ScrubbingDict) ScrubbingDict {
 		groupToRedact: 2,
 		regex:         regexp.MustCompile(s),
 	}
+
+	user, err := user2.Current()
```

Comment:
`user2` 😞 Can't we do
```suggestion
	u, err := user.Current()
```

Or does this break something I haven't seen?